java/runtime/src/main/java/org/ray/runtime/AbstractRayRuntime.java

```diff
@@ -215,13 +216,15 @@ public void free(List<UniqueId> objectIds, boolean localOnly) {
 
   @Override
   public RayObject call(RayFunc func, Object[] args, CallOptions options) {
+    Preconditions.checkState(!workerContext.getCurrentDriverId().isNil());
```

Comment:
we can put this assert in `RayletClientImpl.submit` to avoid duplicating the code.